Extractor/Regions.h

```diff
@@ -1,60 +1,68 @@
 #pragma once
 
+#include <vector>
 #include <string>
 
 /// Fatal Frame 2 constants
 inline const int GameTitleIdLength = 0xB;
-inline const int US_JP_GAME_TITLE_OFFSET = 0x8A010;
-inline const int EU_GAME_TITLE_OFFSET = 0x89810;
-
-enum eRegion {
-  UNKNOWN,
-  NTSCU,
-  NTSCJ,
-  PAL
-};
+//inline const int US_JP_GAME_TITLE_OFFSET = 0x8A010;
+//inline const int EU_GAME_TITLE_OFFSET = 0x89810;
 
-struct GameRegion {
-  eRegion Region;
-  std::string TitleId;
-  int NumFiles;
-  int FileTableStartAddress;
-  int FileTypeTableStartAddress;
-  int FileArchiveStartAddress;
+enum enumGameType
+{
+  GAME_ZERO_UNKNOWN = 0,
+  GAME_ZERO_1,
+  GAME_ZERO_2,
+  GAME_ZERO_3
 };
 
-inline GameRegion UnknownRegion {
-    UNKNOWN,
-    "",
-    0,
-    0,
-    0,
-    0
+enum enumGameRegion
+{
+  REGION_UNKNOWN = 0,
+  REGION_NTSCU,
+  REGION_NTSCJ,
+  REGION_PAL,
+  NUM_REGION
 };
 
-inline GameRegion UsRegion {
-    NTSCU,
-    "SLUS_207.66",
-    0x106B,
-    0x2F90B8,
-    0x3055C0,
-    0x30D40000
+struct ZeroGameLookupData
+{
+  std::string GameTitle;
+  enumGameType GameType;
+  enumGameRegion Region;
+  unsigned int GameTitleOffset;
+  unsigned int NumFiles;
+  unsigned int FileTableStartAddress;
+  unsigned int FileTypeTableStartAddress;
+  unsigned int FileArchiveStartAddress;
 };
 
-inline GameRegion JpRegion {
-    NTSCJ,
-    "SLPS_253.03",
-    0x106B,
-    0x002F85F8,
-    0x304B00,
-    0x30D40000
-};
+const std::vector<ZeroGameLookupData> GAME_LOOKUP_DATA = {
```

Comment:
no need of it being a vector since we already know the size of the array on compilation